app/assets/javascripts/govuk_publishing_components/visual-regression.js

```diff
@@ -0,0 +1,56 @@
+window.GOVUK = window.GOVUK || {}
+
+window.GOVUK.VisualDiffTool = function () {
+  var existingIframe = document.getElementById('visual-diff');
+  if (existingIframe) {
+    existingIframe.parentNode.removeChild(existingIframe);
+    document.body.setAttribute('style', '');
+  } else {
+    var iframe = document.createElement('iframe');
+    iframe.id = 'visual-diff';
+    iframe.style = [
+      'width: 100%;',
+      'height: ' + document.body.scrollHeight + 'px;',
+      'position: absolute;',
+      'top: 0;',
+      'pointer-events: none;',
+      'border: 0;',
+      'mix-blend-mode: difference;',
+      'z-index: 999 !important',
+    ].join(' ');
+
+    iframe.src = _processComparisonURL(_getWindowLocation());
+
+    if (iframe.src) {
+      document.body.setAttribute('style', 'filter: invert()');
+      document.body.appendChild(iframe);
+      console.log("comparing to " + iframe.src);
+    }
+  }
+};
+
+var _processComparisonURL = function (url) {
```

Comment:
Nitpick: space between function and (url) but not elsewhere. Not sure what our convention is, but inconsistent here.

Reply:
Should probably run the JS through https://standardjs.com/